lib/galaxy/webapps/galaxy/api/configuration.py

```diff
@@ -2,19 +2,19 @@
 API operations allowing clients to determine Galaxy instance's capabilities
 and configuration settings.
 """
+import json
+import logging
+import os
 
-from galaxy.web import _future_expose_api_anonymous_and_sessionless as expose_api_anonymous_and_sessionless
-from galaxy.web import _future_expose_api as expose_api
-from galaxy.web import _future_expose_api as expose_api_anonymous
```

Comment:
You caught a bug here but didn't note it - clearly this shouldn't cause any regressions but I guess I'll ping @erasche in case he wants to try to make this anonymously exposed again.

Reply:
Yep, I tried to change this to:
```
from galaxy.web import _future_expose_api_anonymous as expose_api_anonymous
```
but doing a `GET /api/whoami` while not logged in results in an error because `current_user` is `None`.

Reply:
No, need to expose it anonymously. Thanks for the ping @jmchilton